hack/crd-schema-checker.sh

```diff
@@ -16,6 +16,7 @@ for crd in config/crd/bases/*.yaml; do
     mkdir -p "$(dirname "$TMP_DIR/$crd")"
     git show "$BASE_REF:$crd" > "$TMP_DIR/$crd"
     $CHECKER check-manifests \
+        --disabled-validators=NoBools,NoNewRequiredFields \
```

Comment:
I strongly disagree to disable NoNewRequiredFields as we learned in the hard way that adding new required field will block update on the customer side.

Reply:
Just saw this comment.  Will address it on the next push.